test/parked-pages.js

```diff
@@ -58,5 +88,246 @@ describe('Parked Pages', function() {
+    this.timeout(20000);
+    apos5 = await t.create({
+      root: module,
+      modules: {
+        '@apostrophecms/i18n': {
+          options: {
+            locales: {
+              en: {
+                label: 'English',
+                hostname: 'en'
+              },
+              fr: {
+                label: 'French',
+                hostname: 'fr'
+              }
+            }
+          }
+        },
+        '@apostrophecms/page': {
+          options: {
+            park: [
+              ...park2,
+              {
+                slug: '/default3',
+                parkedId: 'default3',
+                _defaults: {
+                  type: 'default-page',
+                  title: 'Default 3'
+                },
+                _children: [
+                  {
+                    slug: '/default3/child1',
+                    parkedId: 'default3child1',
+                    _defaults: {
+                      type: 'default-page',
+                      title: 'Default 3 Child 1'
+                    }
+                  }
+                ]
+              }
+            ]
+          }
+        },
+        'default-page': {}
+      },
+      shortName: apos4.options.shortName
+    });
+    await validate(apos5, [ '/', '/archive', '/default1', '/default2', '/default3', '/default3/child1' ]);
+  });
+});
+
+it('nested parked children work across locales if locales are present from the start', async function() {
+  this.timeout(20000);
+  apos6 = await t.create({
+    root: module,
+    modules: {
+      '@apostrophecms/i18n': {
+        options: {
+          locales: {
+            en: {
+              label: 'English',
+              hostname: 'en'
+            },
+            fr: {
+              label: 'French',
+              hostname: 'fr'
+            }
+          }
+        }
+      },
+      '@apostrophecms/page': {
+        options: {
+          park: [
+            ...park2,
+            {
+              slug: '/default3',
+              parkedId: 'default3',
+              _defaults: {
+                type: 'default-page',
+                title: 'Default 3'
+              },
+              _children: [
+                {
+                  slug: '/default3/child1',
+                  parkedId: 'default3child1',
+                  _defaults: {
+                    type: 'default-page',
+                    title: 'Default 3 Child 1'
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      },
+      'default-page': {}
+    }
+  });
+  await validate(apos6, [ '/', '/archive', '/default1', '/default2', '/default3', '/default3/child1' ]);
 });
+
+async function validate(apos, expected) {
+  const locales = Object.keys(apos.i18n.locales);
+  const slugs = await apos.doc.db.distinct('slug', {
+    slug: /^\//
+  });
+  slugs.sort();
+  assert.deepStrictEqual(slugs, expected);
+  const pages = await apos.doc.db.find({
+    slug: /^\//
```

Comment:
Maybe only relevant to the tests but does a parked slug always start with `/`?

Reply:
Yes, only pages can be parked and "slug starts with a slash" is a guarantee we stand by for pages.